diagrams/initial-api-schema-proposal.json

```diff
@@ -0,0 +1,100 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "API Title",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "https://api.server.test/v1"
+    }
+  ],
+  "paths": {},
+  "components": {
+    "schemas": {
+      "form": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "description": "The ID of the form (Autogenerated)"
+          },
+          "name": {
+            "description": "The name of the form",
+            "allOf": [
+              "$ref": "#/components/schemas/translatableText"
+            ]
+          },
+          "submission_email": {
+            "type": "string",
+            "format": "email",
+            "description": "The email address the form is submitted to"
+          },
+          "pages": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/page"
+            }
+          }
+        }
+      },
+      "page": {
+        "type": "object",
+        "properties": {
+          "position": {
+            "type": "integer",
+            "description": "The position of the page in the order - Branching issues?"
+          },
+          "questionText": {
+            "description": "The text of the question",
+            "allOf": [
+              "$ref": "#/components/schemas/translatableText"
+            ]
+          },
+          "questionShortName": {
+            "description": "The short name for the question",
+            "allOf": [
+              "$ref": "#/components/schemas/translatableText"
+            ]
+          },
+          "hintText": {
+            "description": "The hint text for the question",
+            "allOf": [
+              "$ref": "#/components/schemas/translatableText"
+            ]
+          },
+          "questionType": {
+            "description": "The type of question being asked",
+            "allOf": [
+              "$ref": "#/components/schemas/questionTypes"
+            ]
```

Comment:
Yeah this makes sense - `/<form>/page/<id>` make sense. I can imagine it'll be a string auto-generated from the title rather than a numerical ID though for the runner for the URLs to be readable to analytics platforms